src/styles/blocks/card.scss

```diff
@@ -0,0 +1,119 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: $white;
+
+  &__product {
+    display: block;
+    height: 206px;
+
+    &-img {
+      padding: 32px 19px 40px;
+      width: 160px;
+      height: 134px;
+      flex-shrink: 0;
+    }
+  }
+
+  &__name {
+    margin: 0;
+    padding: 0 16px;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+  }
+
+  &__code {
+    color: $secondary;
+    padding: 0 16px;
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
font-weight: 400; is a default browser value. no needs to redeclare it. check other cases